libsync/SyncMsgPacket.cpp

```diff
@@ -80,11 +80,22 @@ void SyncStatusPacket::encode(int64_t _number, h256 const& _genesisHash, h256 co
     prep(m_rlpStream, StatusPacket, 3) << _number << _genesisHash << _latestHash;
 }
 
-void SyncTransactionsPacket::encode(std::vector<bytes> const& _txRLPs)
+void SyncTransactionsPacket::encode(
+    std::vector<bytes> const& _txRLPs, bool const& _enableTreeRouter, uint64_t const& _consIndex)
 {
+    unsigned fieldSize = 1;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'fieldSize' can be reduced.](https://app.codacy.com/app/fisco/FISCO-BCOS/pullRequest?prid=4692530)